### ISSUE TITLE
Bot: New user experience feedback

### ISSUE THREAD

**Issue by madjin** [webaverse/app#1272]
Based on user testing, it's confusing how to start

- There's no welcome message
- Also no getting started link
- `.help` command is a bit daunting, contains crypto-native jargon
  - Need recipes / examples on how or what to use it

More explanation needed when admins invite the bot into the server for baseline understanding and for how to use it. Here are some ideas:

1. When bot is invited into the server, create a private channel with getting started (collab.land does this)
2. Replace `.help` with `.man` (short for manual) for the command list
3. `.help` contain overview explanation, getting started guide, links to examples and use-cases, and FAQ

I'll be working on these issues in the Webaverse docs first which can be repurposed into other places later.

**Comment by Vianvolaeus** [comment]
Agreed with separating the lists - maybe the **.help [topic]** list could be seperated from the full command list - it's currently at the bottom of the list.

Currently .help [topic] gives commands for that topic which are already in the list - doesn't really *help*

Maybe it could give out a brief help / understanding for topic instead. Not sure. 

![image](https://user-images.githubusercontent.com/80061436/119143678-cd007f80-ba3f-11eb-9668-257a4f760319.png)

**Comment by madjin** [comment]
- Help is really well organized, can skim it
- Wants to mint with URL
- "Do I type the question mark as well?"
- tried minting with brackets (.mint[10])
  - do I need to put a space in between?
- "Can I send this to any wallet? or just webaverse sidechain wallets?"
- "Can I transfer this to other chains?"
  - "I assume I have to pay a gas fee"

Only looks like has 1 NFT, not 10 which was minted.

- Huge shoutout to in-game view mode
- Wants WebXR ability to view NFTs
- "This is awesome work!"
  - "Making it super accessible to get people involved"

Transferring

- "Okay I see @user, address.."
- "Can I do @jin or do I have to type full username?"

Unlockables

- "How do you add unlockables, is that in instructions?"
  - Pack?
  - AddFT?

.gets/.sets -h in DMs doesn't give any feedback

- "What's .play, it take me into Webaverse?"
- "Am I the only person in Webaverse, or is it a separate instance?"
- Would be cool to drop a portal in Discord

**Comment by madjin** [comment]
Dumping my notes here, might useful

User testing feedback

The `.help` is daunting, language is crypto-native

For inspiration: Collab.Land creates a private channel + message about getting started

Need voting feature for top.gg

More explanation, like text between topic and commands for `.help [topic]`

Tokens (how to mint tokens)

Bot command for cheat sheet (use-cases)
- how to mint tickets

`.update` to add comments is broken

30 is the limit for # of editions to be minted at once
error message is confusing, says already minted for 40-100 editions

Minting images of tickets doesn't go far enough
Ticket needs to contain event information (secret message)
Not obvious how to do that, or how to distribute tickets (.store / per person)

Idea: Separate `.help` with `.man`

`.help` gives overview explanation + links to examples / getting started guides / case studies / FAQ

Want to understand how to use the bot, baseline things, before inviting it or when its in the server.

If I'm creator and I hold SILK, people who come to event are also non-crypto native, maximum they can do is ETH/BTC...

Scenario: 

1. On website say we're holding event at this place, people click to buy tickets, check how much, description, instructions, then pay (whatever currency).
2. Confirm on website, with bot create tickets for people who pay (got their ETH address), sitting with 30 ETH address + SILK?
3. How to send tickets to each address in bulk?

Instead of ETH address, maybe users enter Discord IDs at checkout?

Really need to dog food this by hosting an event and using the Dicord bot for it
Could create a test event with partner server (MF)

Join discord server, get sent NFT to discord ID if whitelisted?
Could use `.redeem` command to automatically obtain roles

Discord server > add bot > mint tickets
about as far as gotten
people join discord server, how do users get tickets?
would have to do it 1 by 1, which is actually cool in a way like a teller

Some places in `.help` say id and others say NFT id

Can't transfer multiple different NFTs at once

How to add roles to unlock for `.redeem`?
Look at collab.land for tiers

Question asked:
What if someone doesn't want to go to the event?
Person wants to put ticket on Opensea, charge more
How to to transfer ticket to wallet for Opensea?

How to value align people without buying ticket with crypto?

Idea: `.airdrop @role`
